assets/js/components/ExecutionResults/checksUtils.js

```diff
@@ -125,3 +118,50 @@ export const getCheckExpectations = (catalog, checkID) => {
   }
   return [];
 };
+
+export const isTargetHost = (targetType) => targetType === TARGET_HOST;
+export const isTargetCluster = (targetType) => targetType === TARGET_CLUSTER;
+
+export const isExpect = ({ type }) => type === EXPECT;
+export const isExpectSame = ({ type }) => type === EXPECT_SAME;
+
+export const isAgentCheckError = ({ type }) => !!type;
+
+export const getExpectStatements = (expectationList) =>
```

Comment:
I know that it is a bit redundant, but I would go with `getExpectExpectations`. 
This `Statement` word is coming like out of the blue (I know, already implemented in the prev PRs, but it striked me now)
Or maybe have `filterExpectationByType(type, expectationList)`

Reply:
But we call them `expectations`, not `statements`. 
Anyhow, I'm fine with everyone else likes it